source/gui/addonStoreGui/controls/actions.py

```diff
@@ -153,6 +153,13 @@ def _actions(self) -> List[BatchAddonActionVM]:
 				validCheck=lambda aVMs: AddonListValidator(aVMs).canUseUpdateAction(),
 				actionTarget=self._selectedAddons
 			),
+			BatchAddonActionVM(
+				# Translators: Label for an action that cancel install of the selected add-ons
+				displayName=pgettext("addonStore", "Ca&ncel install of selected add-ons"),

```

Comment:
why not 
```suggestion
				displayName=pgettext("addonStore", "&Cancel install of selected add-ons"),
```

Reply:
This will be the same accelerator key as viewing the source code.

Reply:
Thanks - I missed that